src/controllers/carts.controller.js

```diff
@@ -0,0 +1,65 @@
+const cartsService = require("../services/carts.service");
+const { catchAsync } = require("../utils/error");
+
+const createCarts = catchAsync(async (req, res) => {
+  const { quantity, productOptionId } = req.body;
+  const userId = req.user.id;
+
+  if (!quantity || !userId || !productOptionId) {
+    const err = new Error("KEY_ERROR");
+    err.statusCode = 400;
+    throw err;
+  }
+
+  await cartsService.createCarts(quantity, userId, productOptionId);
+
+  return res.status(201).json({
+    message: "SIGNUP_SUCCESS",
```

Comment:
```suggestion
    message: "SIGNUP_SUCCESS",
```
function name이 createCart인데 return message가 SIGNUP_SUCCESS네요 다시 확인 부탁드릴게요.

참고로 create, delete, update에 대한 client return 값은 해당 행위에 대한 성공 여부를 메세지로서 보내줄 수도 있지만, 해당 method에 대한 SQL query가 정상적으로 반영된, 최신화된 결과 값 자체를 반환하는 것이 프론트에서 재사용하기 용이합니다. 해당 내용은 지금은 그렇구나~ 정도의 느낌만 갖고 넘어가시되 추후에 리팩토링할 때 수정되면 좋을 것 같아요!